sencha-workspace/packages/slate-cbl/src/view/teacher/demonstration/EditWindow.js

```diff
@@ -70,40 +92,39 @@ Ext.define('Slate.cbl.view.teacher.demonstration.EditWindow', {
 
                 forceSelection: true,
                 autoSelect: true
-            },
-            {
+            }, {
```

Comment:
I've never seen this style of braces before? I think the line breaks should be the same for opening and closing brace.